clients/src/test/java/org/apache/kafka/clients/consumer/internals/CommitRequestManagerTest.java

```diff
@@ -106,6 +106,9 @@ public class CommitRequestManagerTest {
     private OffsetCommitCallbackInvoker offsetCommitCallbackInvoker;
     private final Metrics metrics = new Metrics();
     private Properties props;
+    private CommitRequestManager.OffsetFetchRequestState offsetFetchRequestState;
```

Comment:
the offsetFetchRequestState and requestState can both stay in the scope of `testOffsetFetchRequestStateToStringBase` no?